packages/sui-bundler/webpack.config.dev.js

```diff
@@ -5,6 +5,17 @@ const LoaderUniversalOptionsPlugin = require('./plugins/loader-options')
 const babelRules = require('./shared/module-rules-babel')
 require('./shared/shims')
 
+let pwd = process.cwd()
+
+// hack for Windows, as process.env.PWD is undefined in that environment
+// https://github.com/mrblueblue/gettext-loader/issues/18
+// Moreover, to have same caseing we need to map it.
+if (process.platform === 'win32') {
+  const [driveLetter, path] = pwd.split(':')
+  pwd = [driveLetter.toLowerCase(), path].join(':')
```

Comment:
is this really needed? I mean, windows isn't case sensitive, right?

just asking

Reply:
webpack is. You get millions of warnings...